mule-artifact-declaration/src/main/java/module-info.java

```diff
@@ -24,6 +24,9 @@
   exports org.mule.runtime.app.declaration.api.fluent;
   exports org.mule.runtime.app.declaration.api.component.location;
 
+  exports org.mule.runtime.app.declaration.internal.utils to
+      org.mule.runtime.artifact;
```

Comment:
is this only for `org.mule.runtime.app.declaration.internal.utils.Preconditions.checkNotNull`? Refactor that code instead